storage/blockchain/src/service/types.rs

```diff
@@ -3,28 +3,18 @@
 //! Only used internally for our `tower::Service` impls.
```

Comment:
Please update this doc and maybe remove the comment in `mod.rs` importing this file.